### ISSUE TITLE
no support for tmux-256color term

### ISSUE THREAD

**Issue by kurktchiev** [blinksh/blink#1166]
### Checklist

- [x ] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [ x] Read the [docs](http://blink.sh/docs/)
- [x ] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Describe the bug
Setting my TERM to tmux-256color is not making my mosh and ssh connections happy as far as UTF-8 and color support goes.

As soon as I switch it to xterm-256color everything works more or less fine, but I use the extra features of tmux-256color and would love to get that working properly.

**Comment by andrius** [comment]
It works both with ssh and mosh on debian buster and tmux installed with linuxbrew. Just check-ed, I am going to share minified dotfiles with clipboard support
![Uploading 017C4813-8CE3-46C2-AF5D-DC38EC0DE980.png…]()

**Comment by kurktchiev** [comment]
I dunno as soon as I switch the term in tmux.conf UTF support disappears and colors go into base16 as far as I can tell, using the Menslo built in font.

Easiest way to see it, split a window. With term=xterm-256color the split line is a smooth line, switch to term=tmux-256color and the line becomes dashes. As per tmux documentation that explicitly happens when UTF-8 support breaks.


-B

**Comment by andrius** [comment]
Everything works. I am preparing a sandbox repo with dotfiles for blink, going to document tmux/vim/clipboad support, so I will share it. Even emoji works

**Comment by kurktchiev** [comment]
@andrius i looked at your dotfiles and you have xterm-256color defined in them not tmux-256color, xterm-256color works fine, the problem comes when you switch over to tmux-256color.

**Comment by andrius** [comment]
Nope, it works with tmux-color256 on debian-buster with this minimal set of dotfiles: https://github.com/andrius/blink-dotfiles

I am also busy with small howto for blink docs

**Comment by kurktchiev** [comment]
the important lines out of your tmux.conf is:
```
set -g default-terminal "tmux-256color"
set -ga terminal-overrides ",tmux-256color:Tc"
```

vs whats in mine
```
set -g default-terminal "tmux-256color"
set -ga terminal-overrides ",*256col*:Tc"
```

The end result is the same. I get dotted pane seperator line when using the above, and a smooth one when using `xterm-256color`

**Comment by andrius** [comment]
It's always tricky unfortunately. I will share Debian-based Dockerfile with dotfiles repo. Both config options in tmux.conf works well for me

**Comment by kurktchiev** [comment]
What font are you using?

**Comment by andrius** [comment]
Pragmata Pro, but that's "mine" version, not one that is shipped with blink, I've purchased it while ago

**Comment by kurktchiev** [comment]
Ok so what happens if you use https://github.com/blinksh/fonts/blob/master/MesloLGS%20NF.css which is what I am using as I need the powerline symbols

**Comment by carloscabanero** [comment]
@kurktchiev Is this related to #1167 ?

**Comment by carloscabanero** [comment]
We have done a couple fixes here on latest 13.5.11. 
We now bundle the whole Pragmata. We were having speed issues before but it seems like those are properly solved.
Blink is now also properly communicating the proper locale to the remote. Before we were communicating only Unicode, but it seems like that may have been causing a few issues with tmux.

Feel free to reopen if any of the original issues persist.
